### DISCUSSION TITLE
How to regenerate the config

### DISCUSSION THREAD

**Discussion by IlSalta** [i3/i3#5611] [Q&A] [answered]
A few nights ago I was editing the i3 config file (I was copying the config of a rice found on r/unixporn) and after saving the file it gave me an error message.  Let's say that as a fool I didn't make a copy of the file from stock and now I don't know how to rearrange everything. Does anyone know where I can find a stock config or how to regenerate it?

**Comment by okraits** [comment, marked as answer]
Normally you will find the default configuration file at the path `/etc/i3/config`.

Another possibility is to delete the configuration file and run `i3-config-wizard`.
